src/host/host.cpp

```diff
@@ -79,15 +84,23 @@ int main(int argc, char *argv[]) {
     RPCHandler::setEnclaveArgs(argc, argv);
 
     try {
+        std::shared_ptr<ThreadFactory> threadFactory =
+            std::shared_ptr<ThreadFactory>(new ThreadFactory());
+        std::shared_ptr<ThreadManager> threadManager =
+            ThreadManager::newSimpleThreadManager(4);
```

Comment:
4 is the # of worker threads (hard coded I guess, could probably make it a flag)